http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfigurationBuilder.java

```diff
@@ -190,8 +237,63 @@ public static OidcJsonConfiguration loadOidcJsonConfiguration(InputStream is) {
         return adapterConfig;
     }
 
-
     public static OidcClientConfiguration build(OidcJsonConfiguration oidcJsonConfiguration) {
         return new OidcClientConfigurationBuilder().internalBuild(oidcJsonConfiguration);
     }
+
+    private static void failIfUnsupportedAttribute(String unsupportedAttributesParameter, OidcJsonConfiguration config) throws IOException {
+        if (unsupportedAttributesParameter == null) {
+            return;
+        }
+        String[] unsupportedAttributes = unsupportedAttributesParameter.split(" ");
+        for (String attributeName : unsupportedAttributes) {
+            switch(attributeName) {
+                case AUTHENTICATION_REQUEST_FORMAT:
```

Comment:
Similar to the scope RFE, we shouldn't need to hardcode information about the unsupported attributes for WildFly. We'll need to add the handling for unsupported attributes in the `elytron-oidc-client` subsystem.